common-components/index.md

```diff
@@ -399,66 +399,39 @@ If you're considering your options and want more advice, get in touch with your
 
-The Enterprise CRM service is constantly evolving, with new capabilities and functionality being incorporated regularly. Teams across DfE are adopting the Enterprise CRM services to:
-- enhance their current CRM solutions
-- benefit from more integration and data sharing
-- make significant savings on development and support
+The Solution Delivery Team within Digital and Technology can help anyone looking to create a CRM. As a team we work through all stages of the development lifecycle; designing, building, supporting and maintaining systems. As a multidisciplinary team we have both technical and project capabilities, and already support a number of CRMs and web applications.
 
-Here are some of the key benefits:
-
-- **Strategic**
-  - Aligns with DfE and wider government strategy
-  - The Enterprise CRM strategic roadmap has a host of capability enhancements to increase functional maturity
-  - System and development knowledge is retained in-house, enabling quicker and more flexible delivery of future capabilities
-- **Technical**
-  - Ready to use solution, with no procurement required
-  - Over time, there will be increased opportunities to integrate with other DfE systems and improve data sharing
-  - There’s no obligation to move from your current platform, the Enterprise CRM services can help enhance and evolve your current solution
-- **Support**
-  - Specialist in-house engineers delivering 2nd and 3rd line support
-  - Operating model integrated with the ServiceNow toolset
-  - Flexible and fast-turnaround change management
-  - Testing and training for technical and user readiness
-- **Cost**
-  - Save money by consolidating support models, exiting from expensive external supplier contracts, and sharing central resources
-  - Lowered cost of total ownership and economies of scale as multiple business areas contribute to maintenance costs
-
-__Documentation__
-
-Read the [Operating Model](https://educationgovuk.sharepoint.com/:w:/r/sites/lveesfa00073/_layouts/15/Doc.aspx?sourcedoc=%7b5E35BFD1-44C0-478B-81AA-368CAEED4CC3%7d&file=SharedCRM_Service_Operating_Model%20v0.1.docx&action=default&mobileredirect=true&CID=D33D9D75-C993-40C8-AAF5-449A95931FD9&wdLOR=c43C81DA6-1EF4-4B25-B4E7-F798DB011FE5) for more information about the solution design, security, availability, support and governance. If you need access to this document, please contact [Matt Morgan](https://eur.delve.office.com/?u=2bea9bd4-72ee-4174-850b-f6531174a2c7&v=work).  
+We have 3 routes to support:
+- SDT can build the end to end system for you
+- We can offer a blended model where the SDT offers part of a scrum team and the rest is made up of colleagues from within the business or funded contractors
```

Comment:
Need to tidy up and/or unpack some of the terminology here - 'the business' isn't a well-understood term outside DDaT.

Reply:
Updated